public/app/plugins/panel/stat/StatPanel.tsx

```diff
@@ -48,6 +48,7 @@ export class StatPanel extends PureComponent<PanelProps<Options>> {
         theme={config.theme2}
         onClick={openMenu}
         className={targetClassName}
+        disableWideLayout={true}
```

Comment:
we can pretty easily make this another option in stat panel editor options if we are worried about potential backlash to this change - IMO we can default to new behavior and give people a way back - though I think the new behavior makes slightly more sense provides a better design overall (maybe there are some edge cases I'm missing though 😬)

Reply:
This looks good Nathan! It does look good but I would agree that an option makes sense. I wouldn't be surprised if there is someone relying on current behavior and that might surprise them. That being said, I think this is still good for now 👍